src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -39,7 +39,7 @@ def __init__(self,
 
         self._pose_engine = PoseEngine(self._tfengine)
         self._fall_factor = 60
-        self.pose_confidence_threshold = kwargs['confidence_threshold']
+        self.pose_confidence_threshold = confidence_threshold
```

Comment:
@bhavikapanara you addressed the gist of the issue. Thank you!

A couple of notes:
1. As a best practice, every time we identify a potential weakness in the code and fix it, we should also add test coverage to ensure that the problem is solved correctly and to prevent future regression.
2. This solution introduces other issues that I brought up in the original comment.
Namely that `confidence_threshold` is an object instance attribute already used in the super class.
However since the super-class assumes certain properties for image detection, it is a good time to refactor it.

I will accept this PR and merge and will then open a separate issue to refactor FallDetector and TFImageDetection.